Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Stage 1: Build
+
+FROM node:20-alpine AS build
+
+RUN npm install -g pnpm
+
+WORKDIR /app
+COPY . .
+
+# ? Consider --frozen-lockfile
+RUN pnpm install
+
+# ? Add checks?
+
+# ? Consider --prod
+RUN pnpm run build
+
+
+# Stage 2: Serve
+
+FROM caddy:alpine
+
+COPY --from=build /app/dist /usr/share/caddy
+
+EXPOSE 80
+CMD ["caddy", "file-server", "--root", "/usr/share/caddy"]
```

Comment:
just curiosity, Is caddy good for simple file serving?
I've never used caddy before

Reply:
Its a strong web server, relatively modern and fast, a common alternative for Nginx and Apache.
I have been using it in some places, love it so far.

Regarding the use case, as you see it has a built-in option to serve files from a folder, I believe its a good solution  overall.